apps/api/src/app.ts

```
@@ -473,6 +481,44 @@ export async function createApp(
+              clearMessagingRetryDelay = undefined;
+              resolve();
+            };
+          });
+          clearMessagingRetryDelay = undefined;
+        }
+        if (messagingStopped) return;
+        try {
+          await messaging.initialize?.();
+          return;
+        } catch (error) {
+          console.error(
```

Biome (2.5.8)

[error] Don't use console.

(lint/suspicious/noConsole)